src/app/src/main/kotlin/org/imperial/mrc/hint/controllers/ProjectsController.kt

```diff
@@ -95,6 +101,11 @@ class ProjectsController(private val session: Session,
             @PathVariable("projectId") projectId: Int,
             @RequestParam("note") note: String): ResponseEntity<String>
     {
+        logger.info(LogMetadata(
+                action = "Updating project note",
+                request = Request(request),
+                username = userId()),
```

Comment:
If this is your example case, you'll probably want to include some tags too e.g. `tags=listOf("project","notes")`